rundeck-scripts/deploy

```diff
@@ -23,16 +23,9 @@ fi
 
 priv_key="$RD_OPTION_DEPLOY_KEY"
 if [[ -z "$priv_key" ]]; then
-	read priv_key pub_key <<<$(random_key)
+	read priv_key pub_key <<< $(random_key)
+	RD_OPTION_DEPLOY_KEY=$priv_key
 fi
 
 echo "Deploying $contract with private key $priv_key"
-logcmd docker exec rnode ./bin/rnode \
-	$RD_OPTION_RNODE_LAUNCHER_ARGS \
-	-c /var/lib/rnode-static/rnode.conf \
-	deploy \
-	--phlo-limit 1000000000 \
-	--phlo-price 1 \
-	--private-key $priv_key \
-	$RD_OPTION_RNODE_DEPLOY_ARGS \
-	$contract
+python3.6 $INSTALL_DIR/scripts/deploy.py $priv_key $contract
```

Comment:
This won't work for contract set in cases 2 and 4 in the conditional statement above. In those cases, the contract path is inside container, but this script runs on the host and so paths like `./examples/shortfast.rho` don't exist.

It'd be great to fix this without having to run this script inside container. Having to do so would be hassle: access to virtual environment, proper Python version, etc.

Reply:
https://github.com/rchain/rchain-testnet-node/pull/17/commits/bc3b28032d0d91470bf7a8ec0227050a71ef924e